mpl_volume_viewer.py

```diff
@@ -61,11 +61,11 @@ def __init__(self, volume, spacing=None, cmap=plt.cm.gray,
         self.points = points
         self.index = np.array(volume.shape[:3]) // 2
         # aspect is pixel height over pixel width
-        self.raxes[0].imshow(volume[self.index[0], :, :],
+        self.raxes[0].imshow(volume[self.index[0], :, :], labels_cmap,
```

Comment:
actually, this was an oversight and should be `cmap=cmap` — see the `cmap` argument in the init!